packages/paste-core/utilities/grid/src/Column.tsx

```diff
@@ -88,11 +22,11 @@ const getColumnStyles = (props: ColumnProps): ColumnStyles => {
   }
 
   if (props.offset) {
-    columnStyles.marginLeft = getOffset(props.offset);
+    columnStyles.marginLeft = getColumnOffset(props.offset);
   }
 
   if (props.vertical && !props.offset) {
-    columnStyles.minWidth = getVertical(props.vertical);
+    columnStyles.minWidth = getFlexDirection(props.vertical);
```

Comment:
This feels like it still might clash in the future potentially

Reply:
Yeah. I suspect that will be the case. What about `getStackedColumns`?